iwyu_output.cc

```diff
@@ -1774,29 +1774,45 @@ OutputLine PrintableIncludeOrForwardDeclareLine(
     GetSymbolsSortedByFrequency(line.symbol_counts()));
 }
 
-typedef pair<int, string> LineSortKey;
+enum class LineSortOrdinal {
+  kPrecompiledHeader,
```

Comment:
Hmm, I'm also unsure about the naming of enum class enumerators. I think the `k` prefix has outlived its usefulness here, as the names are scoped in `LineSortOrdinal` anyway. I vote to skip the k-prefix.

Reply:
I totally agree. Was just trying to follow existing convention. I'll update and squash.